docs/geedocs/5.2.4/answer/7160004.html

```diff
@@ -79,14 +97,14 @@ <h5>
             </td>
           </tr>
           <tr>
-            <td>881</td>
-            <td>Error Saving Map Layer with Icon</td>
+            <td>946</td>
+            <td>Panoramic Streetview JS error</td>
             <td>
-              The registry of image decoders in the third-party skia library,
-              used in handling icons for map layers, is now being populated.
-              Saving map layers with icons now works.
+              The codes used in two .js files were from different versions of the library.
+              Code was changed to be consistent with the other dependinding .js.
```

Comment:
spelling of "dependinding" - is "depending" the correct word here?
Also, end with "the other depending .js files."